website/app/components/doc/page/sidebar.hbs

```diff
@@ -31,8 +31,22 @@
       </ul>
     </nav>
     {{#if this.structuredPageTree}}
+      <div class="doc-page-sidebar__filter">
+        <Doc::Form::Filter
+          @placeholder="Filter sidebar"
+          autocomplete="off"
+          autocorrect="off"
+          autocapitalize="none"
+          spellcheck="false"
+          @onInput={{this.filterPageTree}}
+        />
+      </div>
```

Comment:
I believe the conditional statement sould start after the filter input, otherwise, when we don't have results it gets removed too and we may end up with blank sidebar.
```suggestion
      <div class="doc-page-sidebar__filter">
        <Doc::Form::Filter
          @placeholder="Filter sidebar"
          autocomplete="off"
          autocorrect="off"
          autocapitalize="none"
          spellcheck="false"
          @onInput={{this.filterPageTree}}
        />
      </div>
    {{#if this.structuredPageTree}}
```

Then we can also have an else for this if, where we tell users that we couldn't find any relevant items.

Reply:
Fixed (in a slightly different way).